### PR TITLE
Add get method to api-routes

### PR DESCRIPTION
**Pull request by ngl4**
All the editing I have done are committed out separately and so they are easier to follow and for y'all to check!

### CODE REVIEW

File: models/item.js [hunk 1]
```diff
@@ -1,19 +1,25 @@
-module.exports = function(sequelize, DataTypes) {
+module.exports = function (sequelize, DataTypes) {
   var Item = sequelize.define("Item", {
-    name: {
+    itemName: {
       type: DataTypes.STRING,
       allowNull: false,
+      validate: {
+        len: [1]
```
Comment (gkpeel) [inline models/item.js:7]: I believe this form of validation will limit the User's item name to a single letter, right? `min: 1` would at least make it only character but I think having `allowNull: false` should be enough to make sure it isn't blank.

(Unless you were trying to do something else that I'm missing)

File: models/item.js [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
+      }
     },
-    reason: {
+    itemReason: {
       type: DataTypes.TEXT,
       allowNull: true,
+      validate: {
+        len: [1]
```
Comment (gkpeel) [inline models/item.js:14]: Same as above

File: models/item.js (hunk 1 above)
Comment (ngl4) [reply models/item.js:7]: true, I think I can remove that.

File: models/item.js (hunk 2 above)
Comment (ngl4) [reply models/item.js:14]: ok will do the same as above

**Pull request closed without merging**